modules/gltf/editor/editor_scene_importer_gltf.h

```diff
@@ -30,13 +30,18 @@
 
 #ifndef EDITOR_SCENE_IMPORTER_GLTF_H
 #define EDITOR_SCENE_IMPORTER_GLTF_H
-
 #ifdef TOOLS_ENABLED
 
+#include "modules/gltf/gltf_document_extension.h"
+
+#include "modules/gltf/gltf_state.h"
+
+#include "core/variant/typed_array.h"
```

Comment:
There's no typed array in this class that I can see? You really need to fix whatever IDE extension you have that keeps doing this, I can't always do time consuming reviews on unnecessary include changes in your PRs...